files/SZWESAMP/ZWEIAPF

```diff
@@ -0,0 +1,32 @@
+//ZWEIAPF JOB
+//*                                                                    
+//* This program and the accompanying materials are made available     
+//* under the terms of the Eclipse Public License v2.0 which          
+//* accompanies this distribution, and is available at                 
+//* https://www.eclipse.org/legal/epl-v20.html                         
+//*                                                                    
+//* SPDX-License-Identifier: EPL-2.0                                   
+//*                                                                    
+//* Copyright Contributors to the Zowe Project. 2020, 2020             
+//*                                                                    
+//*********************************************************************
+//*
+//*  This JCL is used to set APF for the two datasets of Zowe
+//*  Which need it. You can issue this, or use another
+//*  Way to accomplish the task.
+//*
+//*********************************************************************
+//*
+//EXEC14   EXEC  PGM=IEFBR14
+//*
+//*  This dataset holds the APF portion of Zowe
+//         SET LOADLIB='{zowe.setup.dataset.authLoadlib}'
+//*
+//*  This dataset holds product plugins for ZIS,
+//*  ZIS is located in the LOADLIB.
+//         SET PLUGLIB='{zowe.setup.dataset.authPluginLib}'
+//*
+//APFLOAD  COMMAND 'SETPROG APF,ADD,DSN=&LOADLIB' 
```

Comment:
I did not find a way how to use symbolic parameters in COMMAND statement.
It seems to be intended for static command text only.
We can delete SETs, move the libs into COMMANDs, let it replace, but it is needed to check/proceed JCL string continuation.